test/InquirerUnitTest/Helpers/ConsoleKeyInfoFactory.cs

```diff
@@ -23,6 +21,8 @@ public ConsoleKeyInfo Get(char key)
                     return new ConsoleKeyInfo('\0', ConsoleKey.D6, true, false, false);
                 case 'ü':
                     return new ConsoleKeyInfo('ü', ConsoleKey.U, false, false, false);
+                case '@':
+                    return new ConsoleKeyInfo('@', ConsoleKey.D2, false, false, false);
```

Comment:
I think the first boolean parameter represent the `shift` key, and in this case it should be true. Could you check please?

Reply:
You're right the parameter shift needed to be true. Can you please review again?
Thanks!